src/internal/xml-parser.ts

```diff
@@ -263,6 +269,72 @@ export function parseTagging(xml: string) {
   return result
 }
 
+type UploadID = unknown
+
+export type ListMultipartResult = {
+  uploads: {
+    key: string
+    uploadId: UploadID
+    initiator: unknown
+    owner: unknown
+    storageClass: unknown
+    initiated: unknown
+  }[]
+  prefixes: {
+    prefix: string
+  }[]
+  isTruncated: boolean
+  nextKeyMarker: string
+  nextUploadIdMarker: string
+}
+
+// parse XML response for listing in-progress multipart uploads
+export function parseListMultipart(xml: string): ListMultipartResult {
+  const result: ListMultipartResult = {
+    prefixes: [],
+    uploads: [],
+    isTruncated: false,
+    nextKeyMarker: '',
+    nextUploadIdMarker: '',
+  }
+
+  let xmlobj = parseXml(xml)
+
+  if (!xmlobj.ListMultipartUploadsResult) {
+    throw new errors.InvalidXMLError('Missing tag: "ListMultipartUploadsResult"')
+  }
+  xmlobj = xmlobj.ListMultipartUploadsResult
+  if (xmlobj.IsTruncated) {
+    result.isTruncated = xmlobj.IsTruncated
+  }
+  if (xmlobj.NextKeyMarker) {
+    result.nextKeyMarker = xmlobj.NextKeyMarker
+  }
+  if (xmlobj.NextUploadIdMarker) {
+    result.nextUploadIdMarker = xmlobj.nextUploadIdMarker
```

Comment:
```suggestion
 result.nextUploadIdMarker = xmlobj.nextUploadIdMarker || ''
 
 
reason is :
```
TypeError: uploadIdMarker should be of type "string"
    at Client.listIncompleteUploadsQuery (/home/prakash/MinIO/GIT/sdks/minio-js/dist/main/internal/client.js:866:13)
```

Reply:
I validated with S3. as it is not applicable in MinIO